### PR TITLE
Strix Feat - Add RE to Fledging Flight

### PR DESCRIPTION
**Pull request by ammalagonc**
Added Rule Element to calculate Flight Speed based on the Grid Movement rules provided in Player Core 1 (see [link)](https://2e.aonprd.com/Rules.aspx?ID=2355&Redirected=1#:~:text=Normally%2C%20movement%20costs%205%20feet,feet%20of%20movement%20per%20square.)

Added Roll Option per Simon's request.

### CODE REVIEW

**Comment by simonward** [comment]
There are changes to several extra files that should be removed.

File: packs/feats/fledgling-flight.json [hunk 1]
```diff
@@ -29,7 +29,23 @@
...
+                "option": "fledgling-flight",
+                "toggleable": true
+            },
+            {
+                "key": "BaseSpeed",
+                "predicate": [
+                    "fledgling-flight"
+                ],
+                "selector": "fly",
+                "value": "floor(floor(@actor.attributes.speed.total/2)/5)*5"
```
Comment (simonward) [inline packs/feats/fledgling-flight.json:46]:
```suggestion
                "value": "floor(@actor.attributes.speed.total/2)"
```

File: packs/feats/fledgling-flight.json [hunk 2]
```diff
@@ -29,7 +29,23 @@
             "remaster": false,
             "title": "Pathfinder Lost Omens: Ancestry Guide"
         },
-        "rules": [],
+        "rules": [
+            {
+                "alwaysActive": false,
```
Comment (simonward) [inline packs/feats/fledgling-flight.json:34]:
```suggestion
```

**Comment by simonward** [comment]
There's probably also some interactions with "Fully Flighted" and "Juvenile Flight" that need to be handled.

**Comment by ammalagonc** [comment]
Juvenile Flight has an effect that adds additional movement speed to flight. 

Will check for Fully Flighted - probably add a predicate to the roll option to not show up when the feat is added.

**Comment by ammalagonc** [comment]
Made the changes on the other two feats to account for the permutations.

I will close these pull and add a new one and make sure those other files are not added.

**Comment by ammalagonc** [comment]
Made changes on my end -
modified the Juvenile Flight and Fully Flighted. 

I'll close this request and do a new one to account for the changes to those. Will also remove those excess files so they don't keep showing.

**Pull request closed without merging**